packages/celotool/src/e2e-tests/transfer_tests.ts

```diff
@@ -442,8 +443,8 @@ describe('Transfer tests', function(this: any) {
         before(`start geth on sync: ${syncMode}`, () => startSyncNode(syncMode))
 
         describe('Transfer CeloGold >', () => {
-          const GOLD_TRANSACTION_GAS_COST = 29180
-          describe('gasCurrency = CeloGold >', () => {
+          const GOLD_TRANSACTION_GAS_COST = 30005
```

Comment:
Why did this change?

Reply:
It was out of date and being ignored. I adjusted each of the gas costs based on master then added a step step to enforce that they are accurate.